config/locales/en.yml

```diff
@@ -1208,6 +1207,7 @@ en:
     about:
       default_about_text_title: "What is Sharetribe?"
       default_about_text: "This marketplace is powered by Sharetribe platform. With Sharetribe you can easily create your own marketplace website. It's free and only takes a minute. %{click_here_link} to learn more!"
+      default_how_to_use_text: "<h1>How it works</h1><div>Here you can find information about how this site works.</div>"
```

Comment:
Yep, good point!
I somehow didn't stop to think at this point at all, just copied the placeholder there from initial community_customizations row. Definitely good to now put html there!
Thanks for catching!